build-system/release-tagger/utils.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * GitHub API util functions.
+ * TODO: set owner repo defaults
+ * TODO: error handling
```

Comment:
Looks like these are set on lines 32-33 so can these TODOs be removed?

Reply:
Well I want to avoid passing in `{owner, repo, ...}` in the calls every time. octokit has a octokitDefault thing, just haven't seen if it's possible yet.